Tests/NSString+MD5Tests.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  NSString+MD5Tests.swift
+//  PINRemoteImage
+//
+//  Created by Rodrigo Ruiz Murguía on 27/01/22.
+//  Copyright © 2022 Pinterest. All rights reserved.
+//
+
+@testable import PINRemoteImage
+import XCTest
+
+class NSString_MD5Tests: XCTestCase {
+
+    override func setUpWithError() throws {
+    }
+
+    override func tearDownWithError() throws {
+    }
+
+    func testCryptoKitCacheKeyMD5() throws {
+        let testString: NSString = "My awesome test string"
```

Comment:
One additional test that might be useful is ensuring that cryptoKitCacheKeyMD5 returns the same hash as hashCacheKey_removeMeIOS13?

Reply:
done!